src/Server.cpp

```diff
@@ -122,13 +123,29 @@ Server& Server::operator=(const Server& ref)
 void Server::run()
 {
 	RequestParser parser;
+	const int maxEvents = 10;
+	const int timeout = 5000; // epoll timeout in milliseconds
+	const int acceptedIdleTime = 10; // time in seconds until an idle connection is closed
```

Comment:
In #22 I implemented member variables and static const ints for default values. I think this makes it more flexible.
I dont think we would make maxEvents or timeout configurable but I think acceptedIdleTime is a valid configure point

Reply:
I guess it's a more centralised place to store the variables.